### PR TITLE
autoshutdown fix

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8640

### Change description ###

Fix for the autoshutdown issue for AKS

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x  ] No
```

### CODE REVIEW

File: api/idamCheck.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  try {
+    const idamWebUrl = getConfigValue(SERVICES_IDAM_WEB);
+    const axiosInstance = http({} as unknown as Request);
+    const result = await axiosInstance.get(`${idamWebUrl}/o/.well-known/openid-configuration`);
+    if (!result) {
+      console.log('idam must be up to start');
+      process.exit(1);
+    }
+  } catch (err) {
+    console.log('idam must be up to start');
```
Comment (connorpgpmcelroy) [inline api/idamCheck.ts:16]: Do we need the logs?

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with comment

File: api/idamCheck.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/idamCheck.ts:16]: Yes it helps when node fails we need some logs

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**